packages/rum-core/src/common/apm-server.js

```diff
@@ -140,6 +147,29 @@ class ApmServer {
     return new Queue(onFlush, { queueLimit, flushInterval })
   }
 
+  fetchConfig(serviceName, environment) {
+    const serverUrl = this._configService.get('serverUrl')
+    var configEndpoint = `${serverUrl}/config/v1/agents`
+    if (!serviceName) {
+      return Promise.reject(
+        'serviceName is required for fetching central config.'
+      )
+    }
+    configEndpoint += `?service.name=${serviceName}`
+    if (environment) {
+      configEndpoint += `&service.environment=${environment}`
+    }
+    return this._makeHttpRequest('GET', configEndpoint, { timeout: 5000 })
+      .then(response => {
+        return JSON.parse(response)
+      })
+      .catch(reason => {
+        const error = this._constructError(reason)
+        this._loggingService.warn('Failed fetching config!', error)
```

Comment:
we should remove warning from here and only log it on `apm-base.js`. Right now it creates two warnings on the console.